packages/perseus/src/mixins/changeable.ts

```diff
@@ -31,14 +31,9 @@ const USAGE =
  * figured out which way it was called.
  */
 const _changeMultiple = function (
-    component: any,
-    newProps:
-        | string
-        | Record<any, any>
-        | {
-              [key: string]: any;
-          },
-    callback,
+    component: {props: ChangeableProps},
+    newProps: Record<string, any>,
+    callback?: () => unknown,
```

Comment:
Even though this is and "internal" function, can this also be generic?   Something like this? 

```js 
const _changeMultiple = function<Props extends ChagneableProps>(
    component: {props: Props},
    newProps: Partial<Props>,
    callback?: () => unknown
)

```